src/System.Data.Odbc/tests/System.Data.Odbc.Tests.builds

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I did fix the one issue you called out but if there are other things not working let me know and I can help you through them.